### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by JonathanMagnan**
Added website for Entity Framework Plus
Added `System.Linq.Dynamic.Core` package

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -354,6 +354,12 @@ A source generator that can generate strongly-typed-id classes automatically for
 
 [GitHub repository](https://github.com/yangzhongke/LessCode.EFCore.StronglyTypedId)
 
+### Microsoft.EntityFrameworkCore.DynamicLinq
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:357]: Please list supported EF Core versions like it is done for all other extensions on this list.

**Comment by JonathanMagnan** [comment]
Done ;)

**Comment by ErikEJ** [comment]
LGTM

**Review by ajcvickers — APPROVED** [review]

**Comment by ajcvickers** [comment]
Thanks!

**Pull request merged**